Framework/Muon/test/MuonPairingAsymmetryTest.h

```diff
@@ -250,7 +250,7 @@ class MuonPairingAsymmetryTest : public CxxTest::TestSuite {
   void test_that_exec_throws_if_pair_name_is_not_alphanumeric_or_underscored() {
     const std::vector<int> group1 = {1, 2};
     const std::vector<int> group2 = {3, 4};
-    auto ws = createMultiPeriodWorkspaceGroup(2, 1, 10, "pair1");
+    auto ws = createMultiPeriodWorkspaceGroup(2, 1, 10, "pairWS");
```

Comment:
@Mantid-Matthew could you have a look to check this is right?

Reply:
@aybamidele why does this need to change?

Reply:
I have had a look into this and I think it is right, however we should investigate the parallel tests giving false positives for some of the tests (I think that why it sometimes passes)

Reply:
Martin suggested that I do this to fix the failing test.